_layouts/singer.html

```diff
@@ -25,7 +25,21 @@ <h2>2022歌手赛：</h2>
     </div>
 </div>
 
+<!-- Tab links -->
+<div class="tab">
+    <button class="tab-links" onclick="openTab(event, 'singer-list')" id="defaultOpen">大赛介绍</button>
+    <button class="tab-links" onclick="openTab(event, 'sponsor')">赞助商</button>
+</div>
+<hr class="solid">
```

Comment:
Instead of using `hr`, can you simply add `border-bottom: ...` CSS rule for `.tab`?

Reply:
updated.